apps/io-services-cms-webapp/src/utils/azure-user-attributes-manage-middleware-wrapper.ts

```diff
@@ -1,13 +1,11 @@
 import { SubscriptionCIDRsModel } from "@pagopa/io-functions-commons/dist/src/models/subscription_cidrs";
-import {
-  AzureUserAttributesManageMiddleware,
-  IAzureUserAttributesManage,
-} from "@pagopa/io-functions-commons/dist/src/utils/middlewares/azure_user_attributes_manage";
+import { IAzureUserAttributesManage } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/azure_user_attributes_manage";
 import { IRequestMiddleware } from "@pagopa/ts-commons/lib/request_middleware";
 import { IResponse } from "@pagopa/ts-commons/lib/responses";
 import * as E from "fp-ts/lib/Either";
 
 import { BackofficeInternalSubnetCIDRs } from "../config";
+import { AzureUserAttributesManageMiddleware } from "../lib/middlewares/azure_user_attributes_manage-middleware";
```

Comment:
actually I think we could move the functionality within the `AzureUserAttributesManageMiddleware`, but I honestly preferred to avoid making too many changes!

PS: also because I am thinking (sooner or later) of refactoring `AzureUserAttributesManageMiddleware`, avoiding doing the Cosmos query on the CIDRs container when the request comes from the BO (since it is trusted “by default”). Furthermore, to use the `BlockList` class of the Node `net` module, which is precisely a Node core class that allows to check if IP and CIDRs. This way we can get rid of another “legacy” piece of the system! 😉